tests/test_import.py

```diff
@@ -0,0 +1,15 @@
+import jinja2
+import sklearn
+
+
+def test_import():
+	from packaging import version
+	if version.parse(jinja2.__version__) <= version.parse('3.0.3'):
+		raise ImportError('module "Jinja2" must be of a later version than "3.0.3".')
+	if version.parse(sklearn.__version__) < version.parse('1.0.2'):
+		raise ImportError('module "Jinja2" must be version "1.0.2" or later.')
```

Comment:
What is the goal of these checks? From the code above it seems that we do support older versions of these libraries? If we want to work only with versions specified here, it's better to specify this in setup.py.

Reply:
Yeah, the test file is probably not really needed, I omitted it and also slightly changed some other stuff in the PR for https://github.com/eli5-org/eli5/ (the sklearn issue for example was already resolved)